userland/tools/elf2tbf/src/main.rs

```diff
@@ -306,8 +235,9 @@ fn do_work(
 
     // Calculate the offset between the start of the flash region and the actual
     // app code. Also need to get the padding size.
-    let app_start_offset = align8!(header_length);
+    let app_start_offset = align4!(header_length);
```

Comment:
I think this and below needs to still be `align8` as it's the external interface to the program entry point:
http://infocenter.arm.com/help/index.jsp?topic=/com.arm.doc.faqs/ka4127.html

Reply:
Your link points to a document about _stack_ alignment, which I think is unrelated to code entry point

Reply:
You're correct. I was mis-remembering something, I can't find anything about alignment requirements for the PC at entry points. Sorry for the noise